src/plugins/intel_gpu/src/runtime/execution_config.cpp

```diff
@@ -254,6 +254,13 @@ void ExecutionConfig::apply_user_properties(const cldnn::device_info& info) {
     user_properties.clear();
 }
 
+void ExecutionConfig::apply_rt_info(const ov::RTMap& rt_info) {
+    auto runtime_options = rt_info.at("runtime_options").as<ov::RTMap>();
```

Comment:
It looks like that we need to check if "runtime_options" exists as below:
```c++
if (rt_info.find("runtime_options") != rt_info.end())
``` 
Additionally, it always throws an exception in my local tests
```c++
rt_info.at("runtime_options").as<ov::RTMap>();
```

Reply:
Fixed

Reply:
```c++
rt_info.at("runtime_options").as<ov::RTMap>();
``` 
This code still fails with the below exception message on ubuntu 22.04
![image](https://github.com/user-attachments/assets/66605c71-0017-48e9-8ef6-e3d67a1f72b4)
But I'm not sure how to fix it.